apps/ledger-live-desktop/src/renderer/bridge/proxy.js

```diff
@@ -64,6 +66,9 @@ export const getAccountBridge = (
   account: AccountLike,
   parentAccount: ?Account,
 ): AccountBridge<any> => {
+  if (getEnv("EXPERIMENTAL_EXECUTION_ON_RENDERER"))
+    return bridgeImpl.getAccountBridge(account, parentAccount);
+
```

Comment:
```suggestion
  if (getEnv("EXPERIMENTAL_EXECUTION_ON_RENDERER")) {
    return bridgeImpl.getAccountBridge(account, parentAccount);
  }
```

easier to read if it's in more than one line